submissions/laneleka/html-css-popup/css/style.css

```diff
@@ -0,0 +1,275 @@
+html, body {
+ font-family: Arial, Helvetica, sans-serif;
+ height: 100%;
+ width:100%;
+}
+
+.hidden {
+  display: none;
+}
+
+.wrapper-video {
+  max-height: 100vh;
+  height: 100%;
+  width: 100%;
+  overflow: hidden;
+}
+
+header {
+  width: 100%;
+  position: fixed;
+  left: 0;
+  top: 0;
+  z-index: 1;
+  background-color: rgb(33 33 33 / 35%);
+}
+
+.menu {
+  display: flex;
+  justify-content: flex-end;
+  max-width: 1200px;
+  height: 70px;
+  margin: 0 auto;
+}
+
+.menu-items {
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+  margin-left: 15px;
+  outline: none;
+}
+
+li:nth-child(4) {
+  margin-left: 20px;
+}
+
+.menu-link {
+  text-decoration: none;
+  color: #fff;
+  font-size: 16px;
+  text-transform: uppercase;
+  font-weight: bold;
+  padding: 10px;
+}
+
+
+li:focus .menu-link, 
+li:hover .menu-link, 
+li:active .menu-link {
+  color: #F9A7A7;
+  transition: color 0.3s ease-in-out 0s;
+}
+
+.menu-popup {
+  position: relative;
+}
+
+label {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 38px;
+  height: 38px;
+}
+
+.menu-popup-icon {
+  width: 32px;
+  height: 32px;
+  color: #fff;
+  fill: #fff;
+}
+
+.menu-popup:hover .menu-popup-icon, 
+.menu-popup:focus .menu-popup-icon, 
+.menu-popup:active .menu-popup-icon  {
+  width: 38px;
+  height: 38px;
+  color: #4671C6;
+  fill: #F9A7A7;
+  cursor: pointer;
+}
+
+.popup {
+  display: none;
+  position: absolute;
+  top: 54px;
+  right: 0;
+  max-height: 400px;
+  width: 300px;
+  background-color: #ffffff;
+  border-radius: 5px;
+  box-shadow: 2px 2px 1px rgba(41, 52, 73, 0.5);
+  overflow-y: auto; 
+}
+
+.popup-list {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 10px 10px 0 10px;
+}
+
+.popup-list li {
+  margin-left: 0;
+}
+.popup-list li:focus, 
+.popup-list li:hover, 
+.popup-list li:active {
+  background: #eaeaeafa;
+  border-radius: 5px;
+}
+
+.popup-item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 70px;
+  font-size: 14px;
+  text-align: center;
+  color: #4671C6;
+  padding: 10px;
+  text-decoration: none;   
+}
+
+.popup-list li:focus .popup-item, 
+.popup-list li:hover .popup-item, 
+.popup-list li:active .popup-item {
+  color: #F9A7A7;
+  transition: color 0.1s linear;
+}
```

Comment:
For list items in .popup-list-more don't work focus, hover and active pseudo-classes